test/e2e-common.sh

```diff
@@ -159,6 +167,13 @@ function install_knative_serving_standard() {
   wait_until_pods_running knative-serving || return 1
   wait_until_pods_running istio-system || return 1
   wait_until_service_has_external_ip istio-system istio-ingressgateway
+
+  if [[ -n "${INSTALL_MONITORING}" && -n "${INSTALL_MONITORING_YAML}" ]]; then
```

Comment:
Use `(( INSTALL_MONITORING ))` instead of -n

Reply:
Updated to use `(( INSTALL_MONITORING ))` on all checks on INSTALL_MONITORING.